### PR TITLE
Style text descriptions

### PR DESCRIPTION
**Pull request by aattsai**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -190,6 +190,35 @@ body {
   .seventhSegment {
     height: 100vh;
   }
+  .team-photos {
+    margin: 2em !important;
+  }
+ .img-description {
+  position: absolute;
```
Comment (dijonkitchen) [inline css/main.css]: Check indentation.

File: css/semantic.css [hunk 2]
```diff
@@ -11460,7 +11460,7 @@ img.ui.bordered.image {
 .ui.small.images img,
 .ui.small.images svg,
 .ui.small.image {
-  width: 150px;
+  width: 200px;
```
Comment (dijonkitchen) [inline css/semantic.css:11462]: Make a relative %/em/rem for responsiveness?

**Comment by dijonkitchen** [comment]
Mobile view looks OK to me. Only thing is the contact button is lost and the portfolio pictures have a border around them.

**Comment by aattsai** [comment]
I made the borders for portfolio pictures on purpose... I don't know what made the contact button shift to the right though. I couldn't figure it out

**Comment by dijonkitchen** [comment]
Looks like it's there, but you need to scroll right. CSS classes have "right" and "item" have a left margin that's keeping it off the screen.

**Pull request merged**